### PR TITLE
Style update

### PR DESCRIPTION
**Pull request by maciaszczykm**
Style update:

- fixed application header and made its height `66 px`
- made `16 px` gap betwen header and cards
- added underline on replica set list cards name `hover`
- changed `Expose service externally` slide toggle from deploy page to checkbox
- changed `Internal endpoint` format from `host port/protocol` to `host:port protocol`, which is widely used convention

### CODE REVIEW

**Comment by maciaszczykm** [comment]
@bryk PTAL

**Comment by bryk** [comment]
The floating action button on cards list page does not work correctly. Try scrolling down.

**Comment by bryk** [comment]
Also, maybe put each endpoint in a separate line? Now they are not. 
selection_014

**Comment by bryk** [comment]
Looks nice :) PTAL

**Comment by maciaszczykm** [comment]
> The floating action button on cards list page does not work correctly. Try scrolling down.

Oops, forgot to change `absolute` to `fixed`, it should be fine now.

> Also, maybe put each endpoint in a separate line? Now they are not. 

Done.

PTAL

File: src/app/frontend/_variables.scss [hunk 1]
```diff
@@ -24,6 +24,7 @@ $headline-font-size-base:  rem(2.4) !default;
 $subhead-font-size-base:   rem(1.6) !default;
 $body-font-size-base:      rem(1.4) !default;
 $caption-font-size-base:   rem(1.2) !default;
+$toolbar-height-size-base: rem(6.6) !default;
```
Comment (bryk) [inline src/app/frontend/_variables.scss]: Can this be a multiplier of $baseline-grid? Maybe 64px? 2px less is nothign and the toolbar is already large.

File: src/app/frontend/replicasetlist/replicasetlist.scss [hunk 2]
```diff
@@ -16,8 +16,8 @@
 
 .md-fab {
   &.kd-replicaset-deploy {
-    position: absolute;
+    position: fixed;
     right: 0;
-    top: -56px / 2 - $baseline-grid;
+    top: $toolbar-height-size-base - $baseline-grid * 5;
```
Comment (bryk) [inline src/app/frontend/replicasetlist/replicasetlist.scss]: Should this be `$toolbar-height-size-base - /** half of height */ $baseline-grid * 3.5 - /* margin */ $baseline-grid`?
That looks centrally aligned.

**Comment by bryk** [comment]
Please rebase and PTAL.

**Comment by maciaszczykm** [comment]
> Can this be a multiplier of $baseline-grid? Maybe 64px? 2px less is nothign and the toolbar is already large.

Done.

> Should this be $toolbar-height-size-base - /** half of height */ $baseline-grid * 3.5 - /* margin */ $baseline-grid?
That looks centrally aligned.

Done :)

PTAL

**Comment by bryk** [comment]
LGTM

**Pull request merged**